services/app/src/static/style/header.css

```diff
@@ -0,0 +1,54 @@
+#header {
+  width: 100%;
+  height: 80px;
+  top: 0%;
+  left: 0%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #f3f6fa;
+  position: absolute;
+  flex-grow: 1;
+  min-width: 800px;
+}
+#logo {
+  margin-left: 64px;
+}
+#seperator {
+  margin-left: 16.5px;
+  margin-right: 16.5px;
+}
+#roomName {
+  margin-right: 18px;
+}
+#copyLink {
+  margin-right: 20.28px;
+}
+#settings {
+  margin-right: 20.28px;
+}
+#letter {
+  height: 40px;
+  width: 40px;
+  line-height: 40px;
+  right: 110px;
+  border-radius: 30px; /* or 50% */
+  background-color: #ecf0f7;
+  color: #3c64b1;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+  text-transform: uppercase;
+  position: absolute;
+}
+#username {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 16px;
+  color: #212121;
+  position: absolute;
```

Comment:
I see a lot of `absolute` positioning in this stylesheet. We should avoid those as much as possible, as it can be difficult to deal with.

Reply:
I kept absolute positioning for avatar and username because as we minimize the browser window's width from right to left, those two components will shift along. In contrast, if we used relative (or static, or any other way), the components would remain stationary as we drag the window leftwards.

Reply:
Shouldn't be the case if you'd use `display: flex` for the header. It should adapt to the viewport.
Absolute positioning will cause us problems in the future and it should be avoided if possible. In this case it is possible, so lets do it.